metrics/web/docs/features/remote_config/diagrams/remote_config_domain_class.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+
+package base.domain.usecases {
+    interface UseCase<Type, Params> {
+        call: Type
+    }
+}
+
+package common.domain {
+    package repository {
+        interface RemoteConfigRepository {
+            + Future<RemoteConfig> getConfig()
+        }
+    }
+
+    package entities {
+        interface RemoteConfig {
```

Comment:
See the comment above about the name of this class. Also, it seems like this is not an interface but a usual class as all entities we have.